### DISCUSSION TITLE
Adding gudiance inside Ar

### DISCUSSION THREAD

**Discussion by Kingswillwariboko** [google/model-viewer#4276] [Q&A]
How can one add instruction inside the AR experience?

**Comment by elalish** [comment]
Using DOM and the `ar-status` CSS selector, see ar-prompt part of our example: https://modelviewer.dev/examples/augmentedreality/#webXR

This only works in WebXR mode - iOS and Scene Viewer have no customizability here.

**Reply by Kingswillwariboko** [reply]
Thank you Elalish
